### DISCUSSION TITLE
rename the repo to github-act-runner?

### DISCUSSION THREAD

**Discussion by igagis** [ChristopherHX/github-act-runner#26] [Ideas]
The current name `github-actions-act-runner` is somewhat long.
I understand that it combines `github-actions` and `act` to indicate that is actually uses another software, the `act`, to run `github-actions`.

But, maybe combining those `actions` and `act` words into simple `act` would make some sense, since `runner` supposes that it runs _actions_ anyway, nothing else, so the exact word `actions` adds no value to the name, in my opinion.

And name `github-act-runner` is more laconical, yet descriptive.

**Comment by ChristopherHX** [comment]
I understand, removed actions from the repo name.
TODO rename ci builds / update readme

**Reply by igagis** [reply]
also, when running the runner with `--version` option it prints:
```console
github-actions-act-runner version 0.0.0-dev-1c12295ef216ad958768f4d50b4a00ae09b0aa74
```
needs to be corrected also there
